app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -431,8 +431,17 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
         // https://issuetracker.google.com/issues/143570309#comment3
         applicationContext.resources.configuration.uiMode = config.uiMode
 
-        // random StrictMode onDiskRead violation even when Fenix is not running in the background.
-        components.strictMode.resetAfter(StrictMode.allowThreadDiskReads()) {
+        if (isMainProcess()) {
```

Comment:
Not necessarily: I reopened https://github.com/mozilla-mobile/android-components/issues/8639#issuecomment-717553059 if you wanted to call this commit "Issue #whatever: ..."

Reply:
OK, thanks, I think I will go easy on CI and land this, close the ticket manually :).